WordPress/src/main/res/values/strings.xml

```diff
@@ -1059,8 +1065,8 @@
     <string name="error_generic">An error occurred</string>
     <string name="error_moderate_comment">An error occurred while moderating</string>
     <string name="error_edit_comment">An error occurred while editing the comment</string>
-    <string name="error_publish_empty_post">Can\'t publish an empty post</string>
-    <string name="error_publish_no_network">Can\'t publish while there is no connection. Saved as draft.</string>
+    <string name="error_publish_empty_post">Can\'t publish an empty post.</string>
```

Comment:
To be consistent with the rest of our messaging (and Android style guides), I think we should leave this without a period.